recipes/example/meta.yaml

```diff
@@ -9,7 +9,8 @@ package:
 source:
   fn: simplejson-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
-  # md5 is OK but sha256 is desirable.
+  # md5 is OK but sha256 is desirable. On OS X: `shasum -a 256 file-name.tar.gz`,
+  # On Linux: `sha256sum file-name.tar.gz`.
```

Comment:
Would add that `openssl sha256 <file>` works great for me. It supports a variety of checksums. So it is really easy. Also, it comes with the `openssl` package. So, it doesn't require OS specific strategies.